Jenkinsfile

```diff
@@ -49,17 +69,15 @@ pipeline {
     }
 
     stage('Publish Library') {
+      when {
+        expression { isReleaseBranch()  }
+      }
       steps {
         dir(env.REPO_DIR) {
-          sh '''
-             echo "registry=https://registry.npmjs.org" > ./.npmrc
-             echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> ./.npmrc
-          '''
```

Comment:
I believe this is unnecessary after removing the `.npmrc` file from the repo and adding a publishConfig to package.json. Also, I really don't want to end up accidentally committing credentials so it's best to just not put them on disk in the project.